PokedexV2/Views/PokemonFeatures.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  PokemonFeatures.swift
+//  PokedexV2
+//
+//  Created by JonathanA on 19/10/22.
+//
+
+import SwiftUI
+import Kingfisher
+
+struct PokemonFeatures: View {
```

Comment:
Maybe PokemonDetailsView would be a suitable name

Reply:
it's done